en/docs/setup/installing-on-windows.md

```diff
@@ -1,24 +1,17 @@
 # Installing on Windows
 
-!!! tip
-    
-    **Before you begin**, [please see our compatibility
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
-    to find out if this version of the product is fully tested on Windows.
-    
-    WSO2 Identity Server (WSO2 IS) 5.3.0 and 5.4.0 is
-    not supported on Windows 10, due to an issue with Windows 10 and Java.
-    We will update the information about Windows 10 support once the
-    incompatibility is resolved.
-    
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../administer/tested-operating-systems)
```

Comment:
there is no file called "tested-operating-systems" in the administer folder. 
please check